Product Management Service/Code/product-service/src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE products(
+    id int PRIMARY KEY ,
+    name varchar(255),
+    price decimal(10,2),
+    category varchar(255),
+    description text,
+    specifications json,
+    created_timestamp timestamp,
+    updated_timestamp timestamp
+);
+
+
+CREATE TABLE product_images(
+    id int PRIMARY KEY,
+    product_id int,
+    image blob,
+    description text,
+    FOREIGN KEY (product_id) REFERENCES products(id)
```

Comment:
this does not need a created and updated timestamp?? If yes, please change the design and schema accordingly as well.